src/lib/utilities/format-event-attributes.ts

```diff
@@ -35,8 +36,8 @@ const keysToExpand: Readonly<Set<string>> = new Set([
 
 const keysToFormat: Readonly<Set<string>> = new Set(['maximumAttempts']);
 
-export const UnlimitedAttempts = 'Unlimited';
-export const NoExpiration = 'No Expiration';
+export const UnlimitedAttempts = 'unlimited';
+export const NoExpiration = 'no-expiration';
```

Comment:
Do these need a `translate`?
```suggestion
export const UnlimitedAttempts = translate('unlimited');
export const NoExpiration = translate('no-expiration');
```

Reply:
Unfortunately we can't use the `translate` function outside of a svelte component, because our `i18n.init()`  occurs in the root layout, and these files are evaluated before that point. I can try to come up with a solution for this in a future PR.

Reply:
Ah sounds good. Do we want to leave these as is for now then? So it doesn't read as `no-expiration` in the UI?

Reply:
Ahh yes, when I realized `translate()` doesn't work here, I removed the call but didn't change the keys back to their original strings. Good catch!